modules/auxiliary/admin/http/netgear_PNPX_GetShareFolderList_auth_bypass.rb

```diff
@@ -0,0 +1,127 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+    include Msf::Exploit::Remote::HttpClient
+    prepend Msf::Exploit::Remote::AutoCheck
+
+    def initialize(info = {})
+      super(
+        update_info(
+          info,
+          'Name' => 'Netgear PNPX_GetShareFolderList Authentication Bypass',
+          'Description' => %q{
+            This module targets an authentication bypass vulnerability in the mini_http binary of several Netgear Routers
+            running firmware versions prior to 1.2.0.88, 1.0.1.80, 1.1.0.110, and 1.1.0.84. The vulnerability allows
+            unauthenticated attackers to reveal the password for the admin user that is used to log into the
+            router's administrative portal, in plaintext.
+
+            Once the password has been been obtained, attackers can use the exploit/linux/telnet/netgear_telnetenable module
+            to send a special packet to port 23/udp of the router to enable a telnet server on port 23/tcp. The attacker can
+            then log into this telnet server using the new password, and obtain a shell as the "root" user.
+
+            This vulnerability was discovered and exploited by an independent security researcher who reported it to SSD.
+          },
+          'License' => MSF_LICENSE,
+          'Author' => [
+            'Unknown independent researcher', # Vulnerability discovery and PoC creation.
```

Comment:
Can you change the name here to `'Unknown'`. That will match the convention used by other modules and more easily enable us to track metrics on modules that were written by known entities.

Reply:
Sounds good, wasn't sure on formatting of this anyway so thanks for clarifying :)

Reply:
Fixed in https://github.com/gwillcox-r7/metasploit-framework/commit/810fa6be6e3113d440843cbde5d00cb1b131e362